addons/notes/package.json

```diff
@@ -30,6 +31,10 @@
     "marked": "^0.5.1",
     "prop-types": "^15.6.2"
   },
+  "devDependencies": {
+    "@types/jest": "^23.3.9",
+    "@types/react": "^16.7.1"
+  },
```

Comment:
Maybe these 2 @types should be moved into the root repo package.json as well? 
Otherwise they have to be installed for every typescript package

Reply:
makes sense. Can we also configure them in the root `tsconfig`?

Reply:
I moved both types in the root package.json. I also removed prop-types, after adding typescript they were not used anymore. I also moved most of the tsconfig stuff to the root tsconfig